test/README.md

```diff
@@ -2,28 +2,14 @@
 
 ### How to Run
 
-#### TL;DR
+#### Test e2e in dev mode
 
 ```
 make minishift-start
 eval $(minishift docker-env)
-make test-e2e
+make e2e-local
```

Comment:
@corinnekrych we should re-name this target. Perhaps test-e2e-local?

Reply:
done in 
b5dded0